src/_assets/css/main.scss

```diff
@@ -410,27 +413,18 @@ img {
       margin-bottom: 10px;
     }
   }
-  &::-webkit-scrollbar {
-    height: 4px;
-    width: 4px;
-  }
-  &::-webkit-scrollbar-thumb {
-    background: rgba(0,0,0,0.26);
-  }
 }
-#toc {
+#site-toc--side {
```

Comment:
A documentation comment would be helpful here — I don't know what `site-toc--side` is. Alternatively, another name.

Reply:
Adding doc, because the name is from the `_toc.scss` (from Flutter).